srv/components/misc_pkgs/kafka/start.sls

```diff
@@ -26,23 +26,39 @@ Start zoopkeper:
     - name: kafka-zookeeper
     - enable: True
 
-#TODO: find better solution to add delay
-Wait for zookeeper to start:
-  module.run:
-    - test.sleep:
-      - length: 10
+Ensure kafka-zookeeper has started:
+  cmd.run:
+    - name: test 1 -le $(ps ax | grep java | grep -i QuorumPeerMain | grep -v grep | awk '{print $1}' | wc -l)
```

Comment:
Please make use of service command.
If kafka services started as process, utils_setup is going to check the service status, and it may fail there again.

Reply:
@sachinpunadikar the section on line 24 takes care of this requirement.
```
Start zoopkeper:
  service.running:
    - name: kafka-zookeeper
    - enable: True
 ```
Salt ensures that service has actually started in this block.

We are checking if the process has started as additional assurance. Plus, there is another check against endpoint here: https://github.com/Seagate/cortx-prvsnr/pull/1215/files#diff-37b79a5616fb629e878b55f74b7d7dea6b4bc722c54d74b57a6e5bd25964ae6fR20